recipe/conda_build_config.yaml

```diff
@@ -1,2 +0,0 @@
-MACOSX_SDK_VERSION:    # [osx and x86_64]
-  - 10.12              # [osx and x86_64]

```

Comment:
Ohh so we won't need this ? I thought we had to match the version for `MACOSX_SDK_VERSION` and `c_stdlib_version` !